daphne/src/testing.rs

```diff
@@ -300,7 +301,7 @@ impl HpkeDecrypter for MockAggregator {
         task_id: Option<&TaskId>,
     ) -> Result<Self::WrappedHpkeConfig<'s>, DapError> {
         if self.hpke_receiver_config_list.is_empty() {
-            return Err(DapError::fatal("emtpy HPKE receiver config list"));
+            return Err(fatal_error!(err = "emtpy HPKE receiver config list"));
```

Comment:
While we're here ...
```suggestion
            return Err(fatal_error!(err = "empty HPKE receiver config list"));
```